docs/releases/release_notes.md

```diff
@@ -1,14 +1,14 @@
 ### Installed Applications:
 
--   Argo CD [v2.1.5](https://github.com/codefresh-io/argo-cd/releases/tag/v2.1.3)
+-   Argo CD [v2.1.10](https://github.com/argoproj/argo-cd/releases/tag/v2.1.10)
```

Comment:
you don't want to give the tag to our codefresh-io fork directly?

Reply:
Why we don't do it with the other forks?